lib/screens/CameraScreen.dart

```diff
@@ -84,18 +84,18 @@ class _CameraScreenState extends State<CameraScreen> {
       child: Scaffold(
       appBar: AppBar(
         title: Text("Kamera"),
-        automaticallyImplyLeading: false,
+        //automaticallyImplyLeading: false,
```

Comment:
Pokud to tam mít opravdu nechceme, je lepší místo zakomentování smazat celou řádku.